### DISCUSSION TITLE
Why are the errno extern consts declared as long?

### DISCUSSION THREAD

**Discussion by Jules-Bertholet** [jart/cosmopolitan#521] [Q&A] [answered]
The declarations in [libc/errno.h](https://github.com/jart/cosmopolitan/blob/master/libc/errno.h) are all `long`s, even though `errno_t` is `int`. Is there a reason for this?

**Comment by jart** [comment, marked as answer]
We can change that. The storage space it uses is 8 bytes. But it wouldn't matter if we coerce the type to `errno_t`. Want to send a pull request? Please be sure to send the copyright assignment if you do. See CONTRIBUTING.md.

**Reply by jart** [reply]
https://github.com/jart/cosmopolitan/commit/4238e4def9e9c3e72d95529d4c6a1d37658536c9
